tfhe/examples/hpu/bench.rs

```diff
@@ -260,16 +260,19 @@ pub fn main() {
             let roi_start = Instant::now();
 
             let res_hpu = (0..args.iter)
-                .map(|_i| {
+                .filter_map(|i| {
                     let res = HpuRadixCiphertext::exec(&proto, iop.opcode(), &srcs_enc, &imms);
-                    std::hint::black_box(&res);
-                    res
+                    if i == (args.iter - 1) {
+                        Some(res)
+                    } else {
+                        None
+                    }
```

Comment:
Just for the record, there could be a world where a very smart optimizer might trivialize this and not run some perf stuff, but thankfully today it works so all good :)